base/libgit2/libgit2.jl

```diff
@@ -489,10 +489,15 @@ function rebase!(repo::GitRepo, upstream::AbstractString="")
         else
             GitAnnotated(repo, upstream)
         end
+        onto_ann  = if isempty(newbase)
+                        Nullable{GitAnnotated}()
+                    else
+                        Nullable{GitAnnotated}(GitAnnotated(repo, newbase))
+        end
```

Comment:
Or even change the `onto` argument in `GitRebase` to accept a `Union{Void, GitAnnotated}`? I don't think the cost of dynamic dispatch is going to be to burdensome.